powa-web-git/Containerfile

```diff
@@ -8,12 +8,27 @@ RUN apt-get update && apt-get install -y \
     libpq5 \
     libpq-dev \
     git \
-    && git clone https://github.com/powa-team/powa-web.git powa-web.git \
-    && pip3 install psycopg2-binary sqlalchemy tornado \
-    && apt-get purge -y --auto-remove libpq-dev git \
+    curl \
+    && git clone https://github.com/powa-team/powa-web.git powa-web.git --depth=1 \
+    && pip3 install psycopg2-binary sqlalchemy tornado
+
+WORKDIR /usr/local/src/powa-web.git
+
+RUN  \
+    # install node
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash \
```

Comment:
why run random code from the Internet rather than installing a package?
why hardcoding a version, and why this version?

doing this in multiple steps, won't that lead to multiple layers, including very big layers? what is the full image size, including intermediate layers, once this patch is applied? and how much more time does it take to build it?

Reply:
I was wrong thinking that it was better to install ènvm` to get `node` and `npm`  (which is usually the prefered way when doing dev).

Installing things from system packages is much easier and the resulting image is lighter.

Reply:
```localhost/powateam/powa-web-git        latest      270ec0e58940  20 seconds ago  199 MB```

Reply:
As a comparison, it was `204 MB` with the current master.

Reply:
oh that's an excellent news! I was afraid it would cost much more.